platforms/documentation/docs/src/snippets/dependencyManagement/catalogs-toml/groovy/settings.gradle

```diff
@@ -24,7 +24,7 @@ dependencyResolutionManagement {
 
 // tag::change_default_extension_name[]
```

Comment:
This is tagged as `change_default_extension_name` but we are configuring it to the same as the default. That makes this section look a little odd: https://builds.gradle.org/repository/download/Gradle_Release_Check_BuildDistributions/86329189:id/distributions/gradle-8.10-docs.zip!/gradle-8.10-20240802000918%2B0000/docs/userguide/platforms.html#sub:conventional-dependencies-toml

This now reads:
> By default, the libs.versions.toml file will be an input to the `libs` catalog. It is possible to change the name of the default catalog, for example if you already have an extension with the same name:

```
dependencyResolutionManagement {
    defaultLibrariesExtensionName = "libs"
}
```

Reply:
I didn't want to create a new snippet, but it looks like I have no choice.
Thanks for catching this Justin.